ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -453,6 +473,33 @@ class SignalSpec: QuickSpec {
 				observer.sendNext(1)
 				expect(lastValue).to(equal("2"))
 			}
+
+			it("should not keep resulting signal alive indefinitely") {
+				weak var signal: Signal<AnyObject, NoError>? = Signal.never.map { $0 }
```

Comment:
Can we split these out into their own set of tests that don't rely on map and instead have a custom transformer